nix/shells/pkg-sets/js.nix

```diff
@@ -1,17 +1,13 @@
 { pkgs, lib, ... }:
 let
   nodejs = pkgs.nodejs_22;
-  oldYarn = pkgs.yarn.override { inherit nodejs; };
-  yarn = pkgs.yarn-berry.override {
-    inherit nodejs;
-    yarn = oldYarn;
-  };
+  corepack = pkgs.corepack.override { inherit nodejs; };
 in
 {
   packages =
     [
       nodejs
-      yarn
+      corepack
```

Comment:
cool

Reply:
`corepack` is in charge for downloading Node.js package managers so we no longer need to download `yarn` via Nix. This way we can more easily upgrade, simply by changing the `packageManager` field in the root `package.json` file.